rollup.config.dist.js

```diff
@@ -1,15 +1,32 @@
-/* global process require */
+/* global process require Set */
 
+import {readFileSync} from 'fs';
 import node from 'rollup-plugin-node-resolve';
 import commonjs from 'rollup-plugin-commonjs';
 
 var pkg = require('./package.json');
 
+var dependencies = pkg => {
+  var deps = Object.keys(pkg.dependencies || {}).concat(Object.keys(pkg.peerDependencies || {}));
+  return Array.from(new Set(deps.concat(deps.flatMap(dependency => (
+    dependencies(require(`${dependency}/package.json`))
+  )))));
+};
+
 var banner = `/**
  * Fluture bundled; version ${process.env.VERSION || `${pkg.version} (dirty)`}
  */
 `;
 
+var footer = `/** Fluture license
+
+${readFileSync('./LICENSE')}*/
+
+${dependencies(pkg).map(dependency => `/** ${dependency} license
+
+${readFileSync(`./node_modules/${dependency}/LICENSE`)}*/`
```

Comment:
Is it safe to assume that the file is always named __LICENSE__?

Reply:
No. There are several unsafe assumptions in the code above, like the assumption that the dependency is available directly under `node_modules`. However, for my current list of dependencies it works fine. I figured I'd tackle those problems in the unlikely event that they'd come up.

Maybe I should just remove the code completely and manually specify a list of license files to include.

Reply:
Listing the dependencies would give you greater flexibility, but would also create a maintenance problem. I like your current approach provided that there is no risk of bungling a release if and when your assumption is violated.